osm2gtfs/creators/accra/trips_creator_accra.py

```diff
@@ -18,71 +19,83 @@ def add_trips_to_schedule(self, schedule, data):
 
         lines = data.routes
         for route_ref, line in sorted(lines.iteritems()):
-            if type(line).__name__ != "RouteMaster":
+            if not isinstance(line, Line):
                 continue
 
-            line_gtfs = schedule.AddRoute(
-                short_name=line.ref,
-                long_name=line.name.decode('utf8'),
+            line_gtfs = feed.AddRoute(
+                short_name=line.route_id,
+                long_name=line.name,
                 # we change the route_long_name with the 'from' and 'to' tags
                 # of the last route as the route_master name tag contains
                 # the line code (route_short_name)
                 route_type="Bus",
-                route_id=line.id)
-            line_gtfs.agency_id = schedule.GetDefaultAgency().agency_id
+                route_id=line.osm_id)
+            line_gtfs.agency_id = feed.GetDefaultAgency().agency_id
             line_gtfs.route_desc = ""
             line_gtfs.route_color = "1779c2"
             line_gtfs.route_text_color = "ffffff"
 
             route_index = 0
-            for a_route_ref, a_route in line.routes.iteritems():
-                trip_gtfs = line_gtfs.AddTrip(schedule)
+            itineraries = line.get_itineraries()
+            for a_route in itineraries:
+                trip_gtfs = line_gtfs.AddTrip(feed)
                 trip_gtfs.shape_id = TripsCreator.add_shape(
-                    schedule, a_route_ref, a_route)
+                    feed, a_route.route_id, a_route)
```

Comment:
you need to replace `a_route.route.id` by `a_route.osm_id` to keep the shape_id identical.